Tools/LyTestTools/ly_test_tools/o3de/asset_processor_utils.py

```diff
@@ -13,25 +13,12 @@
 import ly_test_tools.environment.process_utils as process_utils
 
 logger = logging.getLogger(__name__)
-processList = ["AssetProcessor_tmp","AssetProcessor","AssetProcessorBatch","AssetBuilder","rc","Lua Editor"]
-
-def start_asset_processor(bin_dir):
```

Comment:
ditto

Reply:
I searched the codebase for any calls of this and it was safe. Unfortunately the other functions in this file had a few usages which is why I didn't remove those.